main.py

```diff
@@ -54,7 +54,7 @@ def upload_file():
         bucket = storage_client.get_bucket("dedbeetz-media")
         blob = bucket.blob('beetz/' + f.filename)
         blob.upload_from_file(f)
-    return ''
+    return 'http://noproblo.dayjo.org/ZeldaSounds/WW_New/WW_Salvatore_Sploosh.wav'
```

Comment:
will this be replaced eventually with an invocation of the file retrieval function? or will we go back to returning '' and route a GET to the retrieval handler as discussed yesterday?

Reply:
sploosh

Reply:
based on your finding of signed urls we won't need a retrieval function after all